batchflow/models/tf/layers/resize.py

```diff
@@ -141,38 +143,42 @@ def subpixel_conv(inputs, factor=2, name='subpixel', data_format='channels_last'
     x = inputs
     with tf.variable_scope(name):
         if layout:
-            x = conv_block(inputs, layout, kernel_size=1, name='conv', data_format=data_format, **kwargs)
+            from .conv_block import ConvBlock
```

Comment:
move this to file beginning

Reply:
@roman-kh told me about recursive import here, but we definitely need some comment in code
on this
And perhaps some layout control so that no one could pass 'B' in layout here

Reply:
Added comment